src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final int bound = 100;
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Color ballColor = colorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(bound);
```

Comment:
Don't create a new instance of 'Random' every time the method is called. Instead, create a single instance as a class-level variable and reuse it.